docs/billing-and-plans/cancel-active-paid-plan.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Cancel Active Paid Subcription Plan
+intro: Sometimes the project needs may change and the current active plan does not fit your project needs. Cancel your currentactive plan to ensure you will not be charged further.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, in the upper-right corner of any page, click your profile icon then click `Billing & plans`
+    > This page shows the current subscription plan information including: plan name, credit limit, remaining credits and expiration date.
```

Comment:
Aren't we adding an extra burden on our docs maintenance by adding notes about attributes that the user will be able to see by herself when she access the web page?
What are the benefits here?
I'd rather keep things simpler, short and straight to the point.

Reply:
```suggestion
```

Agreed, removing the line from doc